src/calculation/MeasureReportBuilder.ts

```diff
@@ -131,12 +132,27 @@ export default class MeasureReportBuilder<T extends PopulationGroupResult> exten
         measureGroup.stratifier.forEach(s => {
           const reportStratifier = <fhir4.MeasureReportGroupStratifier>{};
           reportStratifier.code = s.code ? [s.code] : [];
+          reportStratifier.id = s.id ? s.id : '';
```

Comment:
If there was no code or id on the stratification in the Measure resource, then there shouldn't be in the MeasureReport. Just add them to the stratifier if they exist.